standalone-metastore/metastore-server/src/main/resources/package.jdo

```diff
@@ -345,20 +345,20 @@
       <datastore-identity>
         <column name="CD_ID"/>
       </datastore-identity>
-      <field name="cols" table="COLUMNS_V2" >
```

Comment:
Can you describe the need for this change?

Reply:
Changed to:
```
<field name="cols" persistence-modifier="persistent" table="COLUMNS_V2">
```

Reply:
Just following the directions here:

http://www.datanucleus.org/products/accessplatform/jdo/mapping.html#embedded_collection